src/gui/editors/PianoRoll.cpp

```diff
@@ -4501,6 +4564,21 @@ PianoRollWindow::PianoRollWindow() :
 	DropToolBar *timeLineToolBar = addDropToolBarToTop( tr( "Timeline controls" ) );
 	m_editor->m_timeLine->addToolButtons( timeLineToolBar );
 
+	// -- Note modifier tools
+	// Toolbar
+	QToolButton * noteToolsButton = new QToolButton(m_toolBar);
+	noteToolsButton->setIcon(embed::getIconPixmap("tool"));
+	noteToolsButton->setPopupMode(QToolButton::InstantPopup);
+
+	// Glue
+	QAction * glueAction = new QAction(embed::getIconPixmap("glue"),
+				tr("Glue"), noteToolsButton);
+	connect(glueAction, SIGNAL(triggered()), m_editor, SLOT(glueNotes()));
```

Comment:
I don't know what the other devs think, but do we want to start using the new Qt5 `connect` syntax?

So something one of these:
`connect(glueAction, &QAction::triggered, [=]() { m_editor->glueNotes(); });`
`connect(glueAction, &QAction::triggered, m_editor, &PianoRoll::glueNotes);`

Reply:
When I read this: https://wiki.qt.io/New_Signal_Slot_Syntax the pro would be: `Compile time check of the existence of the signals and slot, of the types, or if the Q_OBJECT is missing.`. Old syntax is not deprecated tho.

I'm OK with both.

Reply:
Yeah, both are fine with me. I'm just curious if moving forward, and with Qt5, we want to use new style or not worry about it.